gammapy/spectrum/utils.py

```diff
@@ -94,63 +94,36 @@ def bin_edges(self, energy):
         return pix1, pix2, energy1, energy2
 
 
-# Todo: find a better place for these functions (Spectrum analysis class?)
-def plot_exclusion_mask(**kwargs):
-    """Plot exclusion mask
+def calculate_predicted_counts(model, aeff, edisp, livetime):
+    """Get npred 
 
-    The plot will be centered at the pointing position
+    The true and reco energy binning are inferred from the provided IRFs.
 
-    Parameters
-    ----------
-    size : `~astropy.coordinates.Angle`
-    Edge length of the plot
-    """
-    from gammapy.image import ExclusionMask
-    from gammapy.spectrum import SpectrumExtraction
-    # Todo: plot exclusion mask as contours with skymap class
-
-    exclusion = ExclusionMask.from_fits(SpectrumExtraction.EXCLUDEDREGIONS_FILE)
-    ax = exclusion.plot(**kwargs)
-    return ax
-
-
-def plot_npred_vs_excess(ogip_dir='ogip_data', npred_dir='n_pred', ax=None):
-    """Plot predicted and measured excess counts
+    TODO: make energy binning optional once `~gammapy.irf.EnergyDispersion`
+    has an evaluate method.
 
     Parameters
     ----------
-    npred_dir : str, Path
-        Directory holding npred fits files
-    ogip_dir : str, Path
-        Directory holding OGIP data
+    model : `~gammapy.spectrum.models.SpectralModel`
+        Spectral model
+    livetime : `~astropy.units.Quantity`
+        Observation duration
+    aeff : `~gammapy.irf.EffectiveAreaTable`
+        EffectiveArea
+    edisp : `~gammapy.irf.EnergyDispersion`, optional
```

Comment:
Here you're saying `edisp` is optional.
I don't think the function currently works without `edisp`!?

Maybe fix this and add a test?
(or leave a TODO for later)